.devcontainer/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.231.6/containers/go/.devcontainer/base.Dockerfile
+
+# [Choice] Go version (use -bullseye variants on local arm64/Apple Silicon): 1, 1.16, 1.17, 1-bullseye, 1.16-bullseye, 1.17-bullseye, 1-buster, 1.16-buster, 1.17-buster
+ARG VARIANT="1.18-bullseye"
```

Comment:
Will this work out of the box for a Windows or Linux x64 user out of the box or must they come in here and change this value? Or does `[Choice]` mean that the user will be presented with an option? If so, can the default just be "1.18" and also can "1.18" be added in the list?

Reply:
I can change it to "1" and it will always pull the latest version of the Go docker container (for version 1 that is). Bullseye works with windows and linux, but if you are an arm mac user I don't think "1" will work right now as they don't seem to have arm64 images for every release yet. I think the best bet for max compatibility and getting most recent Go version would be "1-bullseye"

Reply:
yes, bullseye has both x64 and arm, but the other go images only have x64 still. non-arm people can use the -bullseye image without issue and it will pull the correct image for them automatically.